code/modules/admin/fun_balloon.dm

```diff
@@ -85,6 +119,8 @@
 		M.forceMove(T)
 		to_chat(M, "<span class='notice'>Pop!</span>", confidential = TRUE)
 
+/////////////////////////////Station Crash/////////////////////////////
```

Comment:
Don't do comments like this, they're sort of ugly but also importantly break doc comments (since they start with `///`).

If it's a documentation comment, just do `/// Station Crash`.

Reply:
Here and everywhere else.